### ISSUE TITLE
IndexError: index 0 is out of bounds for axis 0 with size 0

### ISSUE THREAD

**Issue by kevsatt** [XavierTolza/python-timeseries-segmenter#2]
Hey Xavier,

Running your test code gave me this warning:
`C:\Program Files\Python37\lib\site-packages\matplotlib\gridspec.py:335: UserWarning: This figure includes Axes that are not compatible with tight_layout, so results might be incorrect.
  warnings.warn("This figure includes Axes that are not compatible "`

Which is fine since the points still plotted. But when I interact with the points, I'll get these errors:
`Traceback (most recent call last):
  File "C:\Program Files\Python37\lib\site-packages\matplotlib\cbook\__init__.py", line 215, in process
    func(*args, **kwargs)
  File "C:\Program Files\Python37\lib\site-packages\matplotlib\widgets.py", line 1597, in release
    self._release(event)
  File "C:\Program Files\Python37\lib\site-packages\matplotlib\widgets.py", line 1841, in _release
    self.onselect(vmin, vmax)
  File "C:\Program Files\Python37\lib\site-packages\dataframesegmenter\segmenter.py", line 146, in onselect
    self.plot()
  File "C:\Program Files\Python37\lib\site-packages\dataframesegmenter\segmenter.py", line 88, in plot
    class_index = self.class_index(class_name)
  File "C:\Program Files\Python37\lib\site-packages\dataframesegmenter\segmenter.py", line 58, in class_index
    index = np.where(self.classes == class_name)[0][0]
IndexError: index 0 is out of bounds for axis 0 with size 0`

Any help deciphering this would be much appreciated!

**Comment by XavierTolza** [comment]
Hi. Hard to tell for the first problem since I do not have a Windows machine, I suspect there is something to do with your python version or your packages. I updated to python 3 and added version numbers to requirements.txt, can you retry now, updating your packages?
For the second problem, there might be something with your dataset. Does the demo.py script run?
If it does, provide me a copy of your data? It must have a "class" column.

**Comment by kevsatt** [comment]
Hey Xavier, thanks for the reply. I tried on MacOS on Python 3.7 and it runs fine. Really strange. I was using demo.py for both tests.

I remade the venv on Windows with the same package versions and Python 3.7 too and that's where I get the errors. I was hoping the code would be OS agnostic since it just uses Numpy, Pandas, and Matplotlib :(

I also verified the tkinter version on both systems, they are the same.
